Makefile

```diff
@@ -276,6 +271,15 @@ jstest: build-addons build-js-native-api-tests build-node-api-tests ## Runs addo
 		$(CI_JS_SUITES) \
 		$(CI_NATIVE_SUITES)
 
+.PHONY: jscov
+jscov:
+	$(RM) -r out/$(BUILDTYPE)/.coverage
+	$(MAKE) coverage-build-js
```

Comment:
I'm not in favor of having an implicit step that requires online access, preferring ifneq->emit-error, like:
https://github.com/nodejs/node/blob/1375af204a30d1885b9e8bf79725cbd223707439/Makefile#L1224-L1249

Reply:
@mhdawson do you have an opinion, this would require updating the coverage build with additional bootstrapping.

In the current approach to coverage we do implicitly install `nyc`. And, at other points in the Makefile we install other npm artifacts:

```make
lint-md-rollup:
	cd tools/node-lint-md-cli-rollup && npm install
```

I have a slight leaning towards installing the `c8` module implicitly:

1. because it means we don't need to update the existing coverage build.
2. it's one less confusing step for users trying to run coverage.

I'll go with whatever the consensus is though.

Reply:
Since the current job already does this for `nyc` I yield my objection.